Buildings/Fluid/HeatExchangers/ThermalWheels/Data/Generic.mo

```diff
@@ -1,37 +1,71 @@
 within Buildings.Fluid.HeatExchangers.ThermalWheels.Data;
-record Generic "Generic data record for variable-speed wheels"
+record Generic "Generic data record for thermal wheels"
```

Comment:
In the class `Buildings.Fluid.HeatExchangers.ThermalWheels.Data.ASHRAE`, please `final haveVariableSpeed=true,` as below. I believe the ASHRAE is only for the wheels with variable speed drive, right?
```Modelica
record ASHRAE = Buildings.Fluid.HeatExchangers.ThermalWheels.Data.Generic(
  final haveVariableSpeed=true,
  senHeatExchangeEffectiveness(
```
With the change, when instantiating it, like in `Buildings.Fluid.HeatExchangers.ThermalWheels.Latent.Validation.SpeedControlled`, uses will not be able to change the value.

Reply:
add it as suggested